client/analytics/report/customers/config.js

```diff
@@ -2,16 +2,63 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
-
-export const showDatePicker = false;
+import { __, _x } from '@wordpress/i18n';
+import { getRequestByIdString } from '../../../lib/async-requests';
```

Comment:
Nitpick, but I think these 2 imports can be put under internal dependencies and we can drop the `../../../` for readability.

```js
/**
 * Internal dependencies
 */
import { getRequestByIdString } from 'lib/async-requests';
import { NAMESPACE } from 'store/constants';
```

Reply:
whoops, I merged before getting to this. I'll address in the next one, thanks